src/modules/dashboard/components/ColonyActions/ColonyActions.tsx

```diff
@@ -164,6 +168,7 @@ const ColonyActions = ({
   const actions = useTransformer(getActionsListData, [
     (data && { ...data, events: uniqueEvents }) || data,
     commentCount?.transactionMessagesCount,
+    oneTxPaymentExtensionData?.oneTxPaymentExtensionAddress,
   ]);
```

Comment:
There's one thing I haven't caught until now.

If we add two (or three, or whatever number) permissions to a user, we show two actions entries in the actions list, but since we only have one action/transaction hash, we direct the user to the same action page, no matter what action they click:

![Screenshot from 2021-02-12 13-48-28](https://user-images.githubusercontent.com/1193222/107765860-3c791b80-6d3b-11eb-8d7b-22f29f6d3629.png)

The above directs the user to the same page, this:
![Screenshot from 2021-02-12 14-04-47](https://user-images.githubusercontent.com/1193222/107765912-5286dc00-6d3b-11eb-9e3f-43da56cbacde.png)

So I think we should only show one action in the action list, but with same merged title that you do for the event title.

So in this case, we would only have one entry, that would have the title:
```
Assign the architure, administration permission in second to b
```

I think the easiest way to achieve this, is to hook into the transformer and if the action type is a permission, and the transaction hash is the same, just leave one in.